flocker/ca/_ca.py

```diff
@@ -400,9 +400,13 @@ class NodeCredential(PRecord):
         credential object.
     :ivar bytes uuid: A unique identifier for the node this certificate
         identifies, in the form of a version 4 UUID.
+    :ivar bytes cluster_uuid: A unique identifier for the cluster this
+        certificate identifies, in the form of a version 4 UUID.
```

Comment:
It might make slightly more sense for both of these to be `UUID` instead of `bytes` since user-code code elsewhere seems to go and construct a `UUID` from the values.